Exiled.API/Features/Respawn.cs

```diff
@@ -15,12 +15,26 @@ namespace Exiled.API.Features
     using Enums;
     using PlayerRoles;
     using Respawning;
+    using UnityEngine;
 
     /// <summary>
     /// A set of tools to handle team respawns more easily.
     /// </summary>
     public static class Respawn
     {
+        private static GameObject ntfHelicopterGameObject;
+        private static GameObject chaosCarGameObject;
+
+        /// <summary>
+        /// Gets the NTF Helicopter's <see cref="GameObject"/>.
+        /// </summary>
+        public static GameObject NtfHelicopter => ntfHelicopterGameObject ??= GameObject.Find("Chopper");
```

Comment:
`??=` should be avoided when working with Unity objects.

Reply:
omg I took this out and then added it back I'm so silly. I'll fix today

Reply:
I forgor...